internal/setup/setup.go

```diff
@@ -475,6 +487,12 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 		if details.Default {
 			release.DefaultArchive = archiveName
 		}
+		switch details.Pro {
+		case proNone, proApps, proFIPS, proFIPSUpdates, proInfra:
+		default:
+			logf("%s: archive %q ignored due to invalid pro value: %s", fileName, archiveName, details.Pro)
+			continue
```

Comment:
Why `continue` instead of what was here previously?

Reply:
There was nothing here previously. Added `continued` because of the note in the spec which said ` if other pro values are provided, Chisel will not raise an error`. I don't think that should be the way forward to be honest. We are returning errors for the silliest of reasons. Thus, I changed this to return an error.

Perhaps we need to discuss this note in the spec.

Reply:
Discussed offline.